workflow-templates/1gp-ci.yml

```diff
@@ -97,13 +135,13 @@ jobs:
       - name: 🗃️ Checkout Repo 🗃️
         uses: actions/checkout@v4
         with:
-          token: "${{ steps.app-token.outputs.token }}"
+          token: '${{ steps.app-token.outputs.token }}'
           ref: ${{ inputs.ref }}
 
       - name: 🛠️ Force Repository Setup 🛠️
         uses: ncino/action-force-setup@v2
         with:
-          dev-hub-auth-url: "${{ REPLACE_DEVHUB_URL_SECRET }}"
+          dev-hub-auth-url: '${{ REPLACE_DEVHUB_URL_SECRET }}'
           ci-ncino-ssh-private-signing-key: ${{ REPLACE_SSH_PRIVATE_SIGNING_KEY_SECRET }}
           ci-ncino-ssh-public-signing-key: ${{ REPLACE_SSH_PUBLIC_SIGNING_KEY_SECRET }}
```

Comment:
In the other files we just define the secret for these. They shouldn't change by project, so we should probably just put the secret in.